lib/iris/tests/experimental/regrid/test_regrid_area_weighted_rectilinear_src_and_grid.py

```diff
@@ -473,7 +473,7 @@ def test_global_data_reduce_res(self):
         src.coord('longitude').guess_bounds()
         dest = _resampled_grid(src, 0.4, 0.3)
         res = regrid_area_weighted(src, dest)
-        self.assertArrayShapeStats(res, (21, 38), 280.137698, 15.998966)
+        self.assertArrayShapeStats(res, (21, 38), 280.484932, 15.831545)
```

Comment:
What caused these value changes?

Reply:
It's because the bounds at the edges of the source grid are constrained due to the clipping, which changes the areas.